src/filebundle.c

```diff
@@ -261,7 +261,7 @@ int fb_scandir ( const char *path, fb_dirent ***list )
     i = 0;
     while (fb) {
       (*list)[i] = calloc(1, sizeof(fb_dirent));
-      strncpy((*list)[i]->name, fb->name, sizeof((*list)[i]->name));
+      memcpy((*list)[i]->name, fb->name, sizeof((*list)[i]->name));
```

Comment:
Those fixes are wrong. Just add "- 1" (minus one) to the length expression to avoid those warnings.